run-test262.c

```diff
@@ -1964,18 +1974,10 @@ void run_test_dir_list(namelist_t *lp, int start_index, int stop_index)
         } else if (stop_index >= 0 && test_index > stop_index) {
             test_skipped++;
         } else {
-            int ti;
-            if (slow_test_threshold != 0) {
-                ti = get_clock_ms();
-            } else {
-                ti = 0;
-            }
-            run_test(p, test_index);
-            if (slow_test_threshold != 0) {
-                ti = get_clock_ms() - ti;
-                if (ti >= slow_test_threshold)
-                    fprintf(stderr, "\n%s (%d ms)\n", p, ti);
-            }
+            int msec = 0;
+            run_test(p, test_index, &msec);
+            if (verbose > 1 || (msec > 0 && msec >= slow_test_threshold))
```

Comment:
I introduced a small bug here, it should have been written as:
```c
if (verbose > 1 || (slow_test_threshold && msec >= slow_test_threshold))
```
...because now it prints a line for every test that takes >= 1 ms to run. I'll fix that shortly.